generic/tests/netperf.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import division
+from builtins import str
+from builtins import range
+from past.utils import old_div
```

Comment:
The vt framework has the `six` package installed instead of this one, so If we have to handle the division operations we'd better use the one in `six` instead.

Reply:
@luckyh I will explore `six` for the division. Thanks for your feedback.

Reply:
@luckyh In module named `six`, not seeing anything related to `division`. I was referring to `https://six.readthedocs.io/`

Reply:
Sorry, I remember it wrong, it should be `__future__`, see the example of https://github.com/avocado-framework/avocado-vt/pull/1452/files

Reply:
@luckyh Ok, noted. Thanks!!